src/middlewared/middlewared/plugins/crypto_/__init__.py

```diff
@@ -21,5 +21,10 @@ async def setup(middleware):
                 'Failed to set certificate for system.general plugin: %s', e, exc_info=True
             )
 
+    try:
+        await middleware.call('core.environ_update', {'REQUESTS_CA_BUNDLE': '/etc/ssl/certs/ca-certificates.crt'})
+    except Exception as e:
```

Comment:
Why not `failure=True`?

Reply:
It does not really disrupt certificate setup process as that reflects to HTTPS always being available for NAS

Reply:
Why can't this variable be put to the systemd unit file?